lld/ELF/ScriptParser.cpp

```diff
@@ -235,6 +236,26 @@ void ScriptParser::readVersionScriptCommand() {
   }
 }
 
+void ScriptParser::readNoCrossRefs(bool to) {
+  expect("(");
+
```

Comment:
Our code style is quite terse. We only blank lines when there is a significant "pause" need.